src/test/java/test/java/jalau/at18/architects/CardYellowTest.java

```diff
@@ -0,0 +1,19 @@
+package test.java.jalau.at18.architects;
```

Comment:
Just noticed this problem, package should start with `jalau.at18`, because of this, tests are not getting found by gradle task.

Please open a new PR to fix this